open-metadata-resources/open-metadata-demos/virtual-data-connector/virtualizer/src/main/java/org/odpi/openmetadata/virtualdataconnector/virtualiser/Virtualizer.java

```diff
@@ -2,20 +2,54 @@
 /* Copyright Contributors to the ODPi Egeria project. */
 package org.odpi.openmetadata.virtualdataconnector.virtualiser;
 
-
-import org.springframework.beans.factory.annotation.Autowired;
```

Comment:
This file should be removed when the virtualizer is moved into the server chassis as all of its code should be in a module under governance-servers